### DISCUSSION TITLE
Tauri or Flutter for RustDesk desktop?

### DISCUSSION THREAD

**Discussion by rustdesk** [tauri-apps/tauri#4137] [Q&A]
I created a poll on my repo, https://github.com/rustdesk/rustdesk/discussions/533.

Originally I was inclined to Tauri, but after discussion with the author of [flutter_rust_bridge/](https://github.com/fzyzcjy/flutter_rust_bridge/), I get more hesitated.

https://github.com/rustdesk/rustdesk/discussions/533#discussioncomment-2756934

**Comment by rodydavis** [comment]
Here is an example with Flutter Web and Tauri:
https://github.com/rodydavis/flutter_tauri
